packages/web-components/src/horizontalbarchart/horizontalbarchart.ts

```diff
@@ -428,11 +431,23 @@ export class HorizontalBarChart extends FASTElement {
 
     let tooltip: any;
 
-    containerDiv
+    const chartTitleDiv = containerDiv.append('div').attr('class', 'chartTitleDiv');
+    chartTitleDiv
+      .append('div')
       .append('span')
       .attr('class', 'chartTitle')
       .text(data?.chartTitle ? data?.chartTitle : '');
 
+    const hideNumber = this.hideRatio === undefined ? false : this.hideRatio;
+
+    const showRatio = this.variant !== Variant.AbsoluteScale && !hideNumber && data!.chartData!.length === 2;
+    const getChartData = () => (data!.chartData![0].data ? data!.chartData![0].data : 0);
+
+    if (showRatio) {
+      const ratioDiv = chartTitleDiv.append('div').attr('role', 'text');
```

Comment:
Where is the data for the image that you shared.
![image](https://github.com/user-attachments/assets/2b53d246-6f79-4091-b0de-63783c39fc8c)

Reply:
Line 448,449